nil/cmd/nil_block_generator/internal/commands/client.go

```diff
@@ -142,14 +142,24 @@ func DeployContract(path, hexKey string, logger zerolog.Logger) (string, error)
 	salt := types.NewUint256(0)
 	payload := types.BuildDeployPayload(bytecode, common.Hash(salt.Bytes32()))
 
-	fee := types.NewFeePackFromGas(100_000)
+	amount := types.Value0
 
 	srv, err := CreateCliService(hexKey, logger)
 	if err != nil {
 		return "", err
 	}
 
-	_, addr, err := srv.DeployContractExternal(types.BaseShardId, payload, fee)
+	var samrtAccountAddress types.Address
+	if err := samrtAccountAddress.Set(samrtAccountAdr); err != nil {
```

Comment:
Variables look the same. Maybe specify the difference in names?
```suggestion
	if err := smartAccountAddress.Set(smartAccountAddrStr); err != nil {
```